testdata/p4tc_samples_outputs/calculator_parser.h

```diff
@@ -89,30 +82,30 @@ void crc32_update(u32 * reg, const u8 * data, u16 data_size, const u32 poly) {
     u32 lookup_value7 = 0;
     u32 lookup_value8 = 0;
     u16 tmp = 0;
-    if (lookup_table != NULL) {
+    if (crc32_table != NULL) {
```

Comment:
I was trying to compile the `calculator` example and came across the following errors:

```
clang -O2 -g -c -D__KERNEL__ -D__ASM_SYSREG_H -DBTF -Wno-unused-value  -Wno-pointer-sign -Wno-compare-distinct-pointer-types -Wno-gnu-variable-sized-type-not-at-end -Wno-address-of-packed-member -Wno-tautological-compare -Wno-unkno
wn-warning-option -Wnoparentheses-equality -I../include --target=bpf -mcpu=probe -c generated/calculator_parser.c -o generated/calculator_parser.o                                                                                     
In file included from generated/calculator_parser.c:1:                                                                                                                                                                                 
generated/calculator_parser.h:85:9: error: use of undeclared identifier 'crc32_table'                                                                                                                                                  
   85 |     if (crc32_table != NULL) {
   ...
```

Seems like the code is using `crc32_table`, but it's never being defined

Reply:
Could you also include tc runtime path in command "-I ../backends/tc/runtime" ?

Reply:
I see it compiling fine on my end 
 clang -O2 -g -c -D__KERNEL__ -D__ASM_SYSREG_H -DBTF -Wno-unused-value  -Wno-pointer-sign -Wno-compare-distinct-pointer-types -Wno-gnu-variable-sized-type-not-at-end -Wno-address-of-packed-member -Wno-tautological-compare -Wno-unknown-warning-option -Wnoparentheses-equality -I ../backends/ebpf/runtime/ -I ../backends/tc/runtime --target=bpf -mcpu=probe -c calculator_parser.c -o calculator_parser.o

Reply:
The problem was that my pna.h didn't have your modification
It compiles fine with it